biopet-tools/src/test/scala/nl/lumc/sasc/biopet/tools/VcfStatsTest.scala

```diff
@@ -378,26 +248,22 @@ class VcfStatsTest extends TestNGSuite with Matchers {
 
     val blah = checkGeneral(record, List())
```

Comment:
nothing, in test code not all names makes sense ;).

I will change it to something else.